### PR TITLE
Add Events Header to StackLayout

### PR DESCRIPTION
**Pull request by AngeloAnolin**
Some events or methods need to be defined for the control as some of them are not documented in the api reference (https://docs.nativescript.org/api-reference/classes/_ui_layouts_stack_layout_.stacklayout). This will help developers know what events are available for this control where they could wire up some code.

### CODE REVIEW

**Comment by rigor789** [comment]
Hi @AngeloAnolin, thanks for the PR!

The reason we are not listing all the events is because there are many that are inherited from `ViewBase/View`. I would rather include information about event inheritance, and how to look for them than to list all possible events like `doubleTap, longPress, swipe, pan, pinch, loaded, and more...` and then add them to every element in the docs - which I don't think is a good idea.

I will keep this open as a reminder to add more information about the even inheritance, but I will not be merging it.

File: content/docs/en/elements/layouts/stack-layout.md [hunk 1]
```diff
@@ -70,6 +70,11 @@ The following example creates a diagonal stack of items with responsive sizes. I
 ```
 <img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/stack_layout_horizontal_align_children.svg" />
 
+## Events
+| Name | Description |
+|------|-------------|
+`tap` | Emitted when the layout is tapped.
```
Comment (ikoevska) [inline content/docs/en/elements/layouts/stack-layout.md]:
```suggestion
| `tap` | Emitted when the layout is tapped. |
```

**Comment by ikoevska** [comment]
> Hi @AngeloAnolin, thanks for the PR!
> 
> The reason we are not listing all the events is because there are many that are inherited from `ViewBase/View`. I would rather include information about event inheritance, and how to look for them than to list all possible events like `doubleTap, longPress, swipe, pan, pinch, loaded, and more...` and then add them to every element in the docs - which I don't think is a good idea.
> 
> I will keep this open as a reminder to add more information about the even inheritance, but I will not be merging it.

@rigor789 it might make more sense to create an issue about adding event inheritance docs? This will let us close this one.

**Comment by rigor789** [comment]
@ikoevska yes! Opened #233

**Pull request closed without merging**